cmd/relayproxy/config/notifier.go

```diff
@@ -5,12 +5,13 @@ import "fmt"
 type NotifierConf struct {
 	Kind NotifierKind `mapstructure:"kind" koanf:"kind"`
 	// Deprecated: Use WebhookURL instead
-	SlackWebhookURL string              `mapstructure:"slackWebhookUrl" koanf:"slackwebhookurl"`
-	EndpointURL     string              `mapstructure:"endpointUrl" koanf:"endpointurl"`
-	Secret          string              `mapstructure:"secret" koanf:"secret"`
-	Meta            map[string]string   `mapstructure:"meta" koanf:"meta"`
-	Headers         map[string][]string `mapstructure:"headers" koanf:"headers"`
-	WebhookURL      string              `mapstructure:"webhookUrl" koanf:"webhookurl"`
+	SlackWebhookURL          string              `mapstructure:"slackWebhookUrl" koanf:"slackWebhookUrl"`
+	MicrosoftTeamsWebhookURL string              `mapstructure:"microsoftteamsWebhookUrl" koanf:"microsoftteamsWebhookUrl"`
```

Comment:
**🎯 suggestion:** ‏Instead of using a new field here I would suggest to re-use [`WebhookURL`](https://github.com/thomaspoignant/go-feature-flag/blob/b3992642c44b312c366e5deacdfb1de451e84e00/cmd/relayproxy/config/notifier.go#L14) that is generic enough to be used in association of the `microsoftteams` kind field.